app/styles/front.scss

```diff
@@ -314,6 +314,8 @@ $autre: #39747B;
 $autre-light: #a9d2d7;
 $autre-lighter: #cce4e7;
 
+
+/* Should be removed */
```

Comment:
Under which circumstances? If it should be done now, do it now. If not, explain when it should be done.